server/services/zwavejs-ui/lib/constants.js

```diff
@@ -0,0 +1,40 @@
+const { DEVICE_FEATURE_CATEGORIES, DEVICE_FEATURE_TYPES, OPENING_SENSOR_STATE } = require('../../../utils/constants');
+
+const CONFIGURATION = {
+  ZWAVEJS_UI_MQTT_URL_KEY: 'ZWAVEJS_UI_MQTT_URL',
+  ZWAVEJS_UI_MQTT_USERNAME_KEY: 'ZWAVEJS_UI_MQTT_USERNAME',
+  ZWAVEJS_UI_MQTT_PASSWORD_KEY: 'ZWAVEJS_UI_MQTT_PASSWORD',
+};
+
+const EXPOSES = {
```

Comment:
For the future, this constant will be very very long... maybe we can split in multiple files, to avoid to long files...

Reply:
Yes, in that case we'll split it in the future no?